data/serializer/src/main/java/tech/pegasys/teku/api/migrated/TotalAttestationReward.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.migrated;
+
+import java.util.Objects;
+
+public class TotalAttestationReward {
+
+  private final long validatorIndex;
+  private final long head;
+  private final long target;
+  private final long source;
+  private final long inclusionDelay;
+
+  public TotalAttestationReward(
+      long validatorIndex, long head, long target, long source, long inclusionDelay) {
+    this.validatorIndex = validatorIndex;
+    this.head = head;
+    this.target = target;
+    this.source = source;
+    this.inclusionDelay = inclusionDelay;
+  }
+
+  public long getValidatorIndex() {
+    return validatorIndex;
+  }
+
+  public long getHead() {
+    return head;
+  }
+
+  public long getTarget() {
+    return target;
+  }
+
+  public long getSource() {
+    return source;
+  }
+
+  public long getInclusionDelay() {
+    return inclusionDelay;
+  }
```

Comment:
has a note saying phase0 only, i think this is likely an optional (not required) value... we'd likely need it optional in the object to make that happen...

Reply:
ah that's right, I missed that part